.github/workflows/comment-issue-due-date.yml

```diff
@@ -0,0 +1,21 @@
+name: Comment Issue Due Date
+
+on:
+  issues:
+    types:
+      - labeled
+    paths:
+      - 'src/helpers/comment-issue-due-date.ts'
+
+jobs:
+  test:
+    if: contains(fromJSON('["Priority&#58; Critical", "Priority&#58; High", "Priority&#58; Medium", "Priority&#58; Low"]'), github.event.label.name)
```

Comment:
I think it would be better to put this logic inside the helper so it's more readable